engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailProcessor.java

```diff
@@ -178,7 +178,7 @@ public void process(final Event event, final List<Endpoint> endpoints, final boo
             recipientsAuthorizationCriterionExtractor.extract(event)
         );
 
-        Log.tracef("[org_id: %s] Sending email notification to connector", emailNotification);
+        Log.debugf("[org_id: %s] Sending email notification to connector", emailNotification);
```

Comment:
Are we logging at info level in prod? Please make sure this won't fill our logs with the email bodies.

Reply:
Yes the current log level for EmailProcessor in prod is INFO